src/features/Login/index.scss

```diff
@@ -1,4 +1,27 @@
-.login_form{
-    margin: 100px auto;
-    width: 500px;
+body{
+    background-color: #f0f2f5;
+}
+
+.wrapper_Login {
+    margin: 0px auto;
+    padding: 70px 140px 60px 140px;
+    .form_left{
+        background-color: #ffffff;
+        padding: 10px 100px 60px 30px;
+        .form_content h1{
```

Comment:
using &_content... to replace .form_content

Reply:
Fixed at commit [fd36968|https://github.com//pull/1]